CoffeeChat/Models/Body.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Body.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 4/29/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - Request Bodies
+
+struct UserSessionBody: Codable {
+
+    let idToken: String
+
+}
+
+struct UserUpdateBody: Codable {
+
+    let clubs: [String]
+    let graduationYear: String
+    let hometown: String
+    let interests: [String]
+    let major: String
+    let pronouns: String
+
+}
+
+struct MatchingBody: Codable {
+
+    let netIDs: [String]
+    let schedule: [DaySchedule]
```

Comment:
maybe a comment on whose schedule this is? i.e. does it belong to one of the users, is a merging of both of the users, is all of the overlaps between the users, etc.